src/main/java/org/eclipse/yasson/internal/serializer/AbstractContainerDeserializer.java

```diff
@@ -193,6 +193,16 @@ protected Object convertNullToOptionalEmpty(Type propertyType, Object value) {
      */
     public abstract void appendResult(Object result);
 
+    /**
+     * Variant for appendResult including the context.
+     *
+     * @param result An instance result of an item.
+     * @param context The deserialization context.
+     */
+    public void appendResult(Object result, Unmarshaller context) {
```

Comment:
OK, no problem, I just did that because it needed less modifications. I can change the original `appendResult` and add the new parameter to all the implementations.

Reply:
Done!